src/modules/CreateDrops/TokenDrop/data.tsx

```diff
@@ -0,0 +1,42 @@
+import { ICheckbox } from '@/common/components/Checkboxes';
+import { HereLogoIcon, MyNearLogoIcon, NearLogoIcon } from '@/common/components/Icons';
+
+/**
+ * Temporary data providers
```

Comment:
Could we add `// TODO` comment in here too? Or if we will remove this file, you can ignore this

Reply:
we will eventually remove this file once integrated with backend so no worries